package.json

```diff
@@ -51,6 +51,7 @@
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react": "^6.24.1",
     "concurrently": "^3.5.1",
+    "copy-webpack-plugin": "^4.6.0",
```

Comment:
Should both `package-lock.json` & `yarn.lock` be updated?

Reply:
Ideally, yes.  Would you mind PR'ing that?  I'll go ahead and merge this.